wgsl/index.bs

```diff
@@ -1909,11 +1906,11 @@ primary_expression
 postfix_expression
   :
   | BRACKET_LEFT short_circuit_or_expression BRACKET_RIGHT postfix_expression
-  | PAREN_LEFT argument_expression_list* PAREN_RIGHT postfix_expression
+  | argument_expression_list postfix_expression
   | PERIOD IDENT postfix_expression
 
 argument_expression_list
-  : (short_circuit_or_expression COMMA)* short_circuit_or_expression
+  : PAREN_LEFT ((short_circuit_or_expression COMMA)* short_circuit_or_expression)? PAREN_RIGHT
```

Comment:
For readability I would have likely have broken it across two productions:

   : PAREN_LEFT PAREN_RIGHT
   | PAREN_LEFT short_circuit_or_expression (COMMA short_circuit_or_expression)*  PAREN_RIGHT

But it's no big deal to me.